src/ontology/mondo.Makefile

```diff
@@ -457,8 +457,13 @@ tmp/mirror-efo.json: #mirror/efo.owl
 sssom:
 	python3 -m pip install --upgrade pip setuptools && python3 -m pip install --upgrade --force-reinstall sssom==0.3.17
 
-tmp/%.sssom.tsv: tmp/mirror-%.json | sssom
+.PHONY: oaklib
+oaklib:
```

Comment:
This is probably a good approach (I think Nico agrees w/ this as well), updating to the latest version of whatever it is that we rely heavily on which changes frequently. For `mondo-ingest`, I've been simply been `pip install --upgrade`ing everything, but I can change that.